suite-native/analytics/src/constants.ts

```diff
@@ -1,3 +1,21 @@
 export enum EventType {
-    SettingsAnalytics = 'settings/analytics',
+    OnboardingCompleted = 'onboarding_completed',
+    ScreenChange = 'screen_change',
+    AppReady = 'app_ready',
+    AssetsSync = 'assets_sync',
+    WatchPortfolioTimeframeChange = 'watch_portfolio/timeframe_change',
+    CreateReceiveAddress = 'create_receive_address',
+    CreateReceiveAddressShowAddress = 'create_receive_address/show_address',
+    AssetDetail = 'asset_detail',
+    AssetDetailTimeframeChange = 'asset_detail/timeframe_change',
+    TransactionDetail = 'transaction_detail',
+    TransactionDetailParameters = 'transaction_detail/parameters',
+    TransactionDetailCompareValues = 'transaction_detail/compare_values',
+    TransactionDetailInputOutput = 'transaction_detail/input_output',
+    TransactionDetailExploreInBlockchain = 'transaction_detail/explore_in_blockchain',
+    SettingsChangeCurrency = 'settings/change_currency',
+    SettingsChangeTheme = 'settings/change_theme',
+    SettingsChangeBtcUnit = 'settings/change_btc_unit',
+    SettingsDiscreetToggle = 'settings/discreet_toggle',
+    SettingsDataPermission = 'settings/data_permission',
```

Comment:
nit: I'd still keep analytics there but if you want to keep it renamed, how about `settings/analytics_data_permission`? Data permission is too general imo and could be also related to local storage etc.

Reply:
Yes, please do not mess with names.